### DISCUSSION TITLE
Move tiled windows with mouse

### DISCUSSION THREAD

**Discussion by TheSunCat** [i3/i3#4856] [Q&A] [answered]
Hi!
I'm switching from Sway to i3, and almost everything works great out of the box.

Currently, I can resize tiled windows with Alt+right click, as defined in my config file with: `floating_modifier Mod1`. However, I'm really missing the feature where tiled windows can be moved to a different tiled position/screen with Alt+drag. Is it possible to do this within i3?

Thanks!

**Comment by Airblader** [comment, marked as answer]
#3085
